script.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * fn - Variable to store the first name entered by the user.
+ * ln - Variable to store the last name enetered by the user.
+ */
+function getFullName() {
+  document.getElementById("fname").addEventListener("keyup", full_name_update);
+  document.getElementById("lname").addEventListener("keyup", full_name_update);
+
+  var fn = document.getElementById("fname");
+  var ln = document.getElementById("lname");
+  /**
+   * This fuction is responsible for updating the full name of the user.
+   */
+  function full_name_update() {
+    document.getElementById("full_name").value = fn.value + " " + ln.value;
+  }
+}
+  /**
+   * This function is responsible for validating the first name entered by the user.
+   * regex - This is the variable for storing the valid pattern.
+   * fnamevalue - This is the variable for storing the first name entered by the user.
+   */
+function validateFirstName() {
```

Comment:
Please remove those functions which you are not using

Reply:
Done.